relayer/pkg/starknet/client.go

```diff
@@ -56,8 +57,9 @@ func NewClient(_chainID string, baseURL string, lggr logger.Logger, timeout *tim
 	}
 
 	client := &Client{
-		Provider: starknetrpc.NewProvider(c),
-		lggr:     lggr,
+		Provider:  starknetrpc.NewProvider(c),
+		EthClient: c,
```

Comment:
add the ethrpc client to the struct so we can take advantage of its batch call capabilities!